src/mlpack/methods/ann/layer/reparametrization.hpp

```diff
@@ -46,10 +46,12 @@ class Reparametrization
    * @param latentSize The number of output latent units.
    * @param stochastic Whether we want random sample or constant.
    * @param includeKl Whether we want to include KL loss in backward function.
+   * @param beta Hyperparameter for constrained variational frameworks.
```

Comment:
'beta' parameter needs to be described properly with reference to the paper. I suggest a small paragraph in the class description and mention here in the function.